src/frontend/pages/report-history.vue

```diff
@@ -44,7 +38,6 @@ export default {
     if (this.id) {
       const index = this.reports.findIndex(e => e.data.createdAt === this.id)
       const reportSelected = this.reports[index]
-      this.reports[index] = reportSelected.reveal = true
       this.$vuetify.goTo(`[id='${reportSelected.id}']`)
```

Comment:
There's no guarantee that `reportSelected` is defined.
Also, I don't understand the use of this redirection.

Reply:
this will be deleted